telemetry/opting-out-of-telemetry.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="opting-out-of-telemetry"]
+= Opting out of Telemetry
+include::modules/common-attributes.adoc[]
+:context: logs-and-pod-status
+
+toc::[]
+
+[role="_abstract"]
+You must comply with privacy laws to opt out of Telemetry.
```

Comment:
Who are 'you'? Which laws?

Reply:
We might need to tell where to find the opt-out toggle in the central configuration UI.

Reply:
It's ok to use the exact same text as the OpenShift docs if needed (and OK'd by legal): https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html/telemetry/opting-out-of-telemetry